src/views/account/Grades/Modals/GradeReaction.tsx

```diff
@@ -0,0 +1,323 @@
+import React, { useEffect, useRef, useState } from "react";

```

Comment:
Faire l'import par rapport à la suggestion d'après :

```suggestion
import React, { useEffect, useLayoutEffect, useRef, useState } from "react";
```